docs/es/week12/12-2.md

```diff
@@ -0,0 +1,323 @@
+---
+lang-ref: ch.12-2
+title: Decoding Language Models
+lecturer: Mike Lewis
+authors: Trevor Mitchell, Andrii Dobroshynskyi, Shreyas Chandrakaladharan, Ben Wolfson
+date: 20 Apr 2020
+translator: GastonMazzei (Gaston Mazzei)
+---
+
+
```

Comment:
As per the [translations guidelines](https://github.com/Atcold/pytorch-Deep-Learning/wiki/Translation-Guidelines), we should preserve the original English content (commented out).

Reply:
Oh! Major thing here! I'm on it, thanks!